pkg/es/textTemplate_test.go

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) 2020 The Jaeger Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package es
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestParse(t *testing.T) {
+
+	str := "tesing parse"
+	writer := new(bytes.Buffer)
+	testTemplateBuilder := TextTemplateBuilder{}
+	parsedStr, err := testTemplateBuilder.Parse(str)
+	require.NoError(t, err)
+	err = parsedStr.Execute(writer, "")
+	require.NoError(t, err)
+	assert.Equal(t, str, writer.String())
+
+}
```

Comment:
I would make this test a little more strict as the `Execute` function could simply just return the same string without applying any template instead of forwarding the call to the template lib. A suggestion might be (feel free to change the example, you get the idea :) ):
```
func TestParse(t *testing.T) {
	const wantYear = 2021
	values := struct {
		Year int
	}{wantYear}

	template := "This year is {{.Year}}."
	writer := new(bytes.Buffer)
	testTemplateBuilder := TextTemplateBuilder{}
	parsedStr, err := testTemplateBuilder.Parse(template)
	require.NoError(t, err)
	err = parsedStr.Execute(writer, values)
	require.NoError(t, err)
	assert.Equal(t, "This year is 2021.", writer.String())
}
```

Reply:
Yep got it. Would be added in next commit